### PR TITLE
Autocomplete migration

### PR DESCRIPTION
**Pull request by barry800414**
to solve #277

### CODE REVIEW

File: src/js/main/index.js [hunk 1]
```diff
@@ -80,3 +80,65 @@ for (let i = 0; i < clear_radio_btn.length; i++) {
...
+        source: function (request, response) {
+            if(ga){
+                ga('send', 'event', 'LANDING_PAGE', 'company-query-autocomplete-search', request.term);
+            }
+            $.ajax({
+                url: WTS.constants.backendURL + 'companies/search',
+                data: { key : request['term'] },
+                dataType: "json",
+            }).done(function( res ) {
+                let nameList = new Array();
```
Comment (mark86092) [inline src/js/main/index.js]: let nameList = [];

File: src/js/main/index.js [hunk 2]
```diff
@@ -80,3 +80,65 @@ for (let i = 0; i < clear_radio_btn.length; i++) {
...
+                console.log(res);
```
Comment (mark86092) [inline src/js/main/index.js]: no console.log

File: src/js/main/index.js [hunk 3]
```diff
@@ -80,3 +80,65 @@ for (let i = 0; i < clear_radio_btn.length; i++) {
...
+
+                res.forEach((item, i) => {
+                    nameList.push({"value": item['name'], "company_id": item.id});
```
Comment (mark86092) [inline src/js/main/index.js]: `{value: item.name, company_id: item.id}`

**Comment by mark86092** [comment]
I did
1. 把 jquery-ui 的圖片，CSS, JS 放入，而且只放入跟 autocomplete 有關的
2. 修改 gulp 設定，避免兩次載入檔案

**Comment by barry800414** [comment]
1. thanks, 原來還有icons
2. 其實不會載入兩次說，我看最後dependencies.js只有一份?

**Comment by mark86092** [comment]
2) 我沒去做交叉比對，不過，既然都列了，就全列吧

**Comment by barry800414** [comment]
I've done my review

**Comment by mark86092** [comment]
I'll merge after #280, please wait it

**Comment by mark86092** [comment]
This branch is rebased onto the newest dev branch

And commit 84aaaf5 is added

**Pull request merged**